src/app.py

```diff
@@ -324,3 +324,21 @@ def remove_contributor(username, submission_name, removed_username):
 def get_contributors(username, submission_name):
     dbw = DatabaseWrapper(GITOLITE_ADMIN_PATH, DATABASE_PORT)
     return dbw.get_contributors(username, submission_name)
+
+
+@app.route('/<username>/projects', methods=['GET'])
+def get_projects(username):
+    dbw = DatabaseWrapper(GITOLITE_ADMIN_PATH, DATABASE_PORT)
+    return dbw.get_projects(username)
```

Comment:
These need to be json format in order to work in flask ( `return jsonify({"projects": dbw.get_projects(username)})`